OpenDreamClient/Interface/DreamInterfaceManager.cs

```diff
@@ -535,6 +536,32 @@ internal sealed class DreamInterfaceManager : IDreamInterfaceManager {
                 // But also have it here in case a local winget ever wants it
                 case "hwmode":
                     return "true";
+                case "windows":
+                    var windowEnum = Windows.Values.GetEnumerator();
+                    if(!windowEnum.MoveNext())
+                        return string.Empty;
+                    List<string> windowIds = new();
+                    do {
+                        if(!((WindowDescriptor)windowEnum.Current.ElementDescriptor).IsPane){
+                            windowIds.Add(windowEnum.Current.Id);
+                        }
+                    } while(windowEnum.MoveNext());
+                    return string.Join(';', windowIds);
+                case "panes":
+                    var paneEnum = Windows.Values.GetEnumerator();
+                    if(!paneEnum.MoveNext())
+                        return string.Empty;
+                    List<string> paneIds = new();
+                    do{
+                        if(((WindowDescriptor)paneEnum.Current.ElementDescriptor).IsPane){
+                            paneIds.Add(paneEnum.Current.Id);
+                        }
+                    } while(paneEnum.MoveNext());
+                    return string.Join(';', paneIds);
```

Comment:
Use `.Where().Select()` instead
```suggestion
                case "windows":
                    return string.Join(';',
                        Windows.Where(pair => !((WindowDescriptor)pair.Value.ElementDescriptor).IsPane).Select(pair => pair.Key));
                case "panes":
                    return string.Join(';',
                        Windows.Where(pair => ((WindowDescriptor)pair.Value.ElementDescriptor).IsPane).Select(pair => pair.Key));
```